docs/notebooks/how_bambi_works.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>A link function, $g$, is...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>